### PR TITLE
HELP.gv.at: Add spider.

### PR DESCRIPTION
**Pull request by Lukas0907**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,6 +31,7 @@ Feeds is currently able to create Atom feeds for the following sites:
   media to libraries based on the biblioweb software.
 * `cbird.at <http://www.cbird.at>`_: Newest releases of the cbird software
 * `falter.at <http://www.falter.at>`_: Newest articles
+* `HELP.gv.at <https://help.gv.at>`_: News and changes in law
```
Comment (nblock) [inline README.rst]: … in austrian law

File: feeds/spiders/HelpGvAt.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import scrapy
+
+from feeds.loaders import FeedEntryItemLoader
+from feeds.loaders import FeedItemLoader
+
+
+class HelpGvAtSpider(scrapy.Spider):
+    name = 'help.gv.at'
+    allowed_domains = ['help.gv.at']
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: you could re-use name here

File: feeds/spiders/HelpGvAt.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            '/content/194/Seite.1940000.html',
+            self._parse_lists)
+
+    def _parse_news_box(self, response):
+        il = FeedItemLoader()
+        il.add_value('title', 'HELP.gv.at')
+        il.add_value(
+            'subtitle',
+            'Ihr Wegweiser durch die Behörden und Ämter in Österreich')
+        il.add_value('link', 'http://help.gv.at')
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: The site offers https, let's use it.

File: feeds/spiders/HelpGvAt.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            yield scrapy.Request(response.urljoin(link), self._parse_item)
+
+    def _parse_item(self, response):
+        remove_elems = [
+            'h1', '.nono', '.acceptance_org', '.state',
+            '.gentics-portletreload-position-notvisibleposition'
+        ]
+        il = FeedEntryItemLoader(response=response,
+                                 timezone=self._timezone,
+                                 base_url='http://{}'.format(self.name),
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: The site offers https, let's use it.

File: feeds/spiders/HelpGvAt.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class HelpGvAtSpider(scrapy.Spider):
+    name = 'help.gv.at'
+    allowed_domains = ['help.gv.at']
+
+    _timezone = 'Europe/Vienna'
+
+    def start_requests(self):
+        yield scrapy.Request(
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: What happens in case this Request takes longer than the others?

**Comment by nblock** [comment]
LGTM, see the comments.

File: feeds/spiders/HelpGvAt.py (hunk 5 above)
Comment (Lukas0907) [inline feeds/spiders/HelpGvAt.py]: Nice catch, it's indeed a race condition.

**Comment by Lukas0907** [comment]
I've updated the branch according to your comments! Thanks for your input.

File: feeds/spiders/HelpGvAt.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+            'icon',
+            'https://www.{}/HLPD_Static/img/favicon.ico'.format(self.name))
+        il.add_value(
+            'logo',
+            'https://www.{}/HLPD_Static/img/120924_Header_helpgv_links.jpg'.
+            format(self.name))
+        yield il.load_item()
+
+        yield scrapy.Request(
+            'https://www.help.gv.at/Portal.Node/hlpd/public'
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: you could use the name here too.

File: feeds/spiders/HelpGvAt.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+            '/content/171/Seite.1710000.html',
+            self._parse_lists)
+
+        yield scrapy.Request(
+            'https://www.help.gv.at/Portal.Node/hlpd/public'
```
Comment (nblock) [inline feeds/spiders/HelpGvAt.py]: you could use the name here too.

**Comment by nblock** [comment]
Thx!

**Pull request merged**